xbmc/cores/VideoPlayer/DVDSubtitles/DVDSubtitleParserSubrip.cpp

```diff
@@ -12,6 +12,8 @@
 #include "cores/VideoPlayer/Interface/TimingConstants.h"
 #include "utils/StringUtils.h"
 
+#include <string>
```

Comment:
needed? Stringutils should include string already

Reply:
for me ok remove it, but nested includes IMO are not always reliable